php/utils.php

```diff
@@ -774,6 +774,29 @@ function trailingslashit( $string ) {
 	return rtrim( $string, '/\\' ) . '/';
 }
 
+/**
+ * Normalize a filesystem path.
+ *
+ * On windows systems, replaces backslashes with forward slashes
+ * and forces upper-case drive letters.
+ * Allows for two leading slashes for Windows network shares, but
+ * ensures that all other duplicate slashes are reduced to a single.
+ * Ensures upper-case drive letters on Windows systems.
+ * Allows for Windows network shares.
```

Comment:
To add my nitpicks, can just remove this comment "Allows for WIndows network shares." (taken from the [since](https://github.com/WordPress/wordpress-develop/blob/4.9.4/src/wp-includes/functions.php#L1706) I see) as it's covered above. Also maybe add `Copied from core wp_normalize_path().` or similar, and `access public` and `category System` tags as in for instance `trailingslashit()`.